server/modules/slack/slackIntegration.js

```diff
@@ -160,3 +168,27 @@ export const addToChannel = async (fellowEmail, channel) => addOrRemove(fellowEm
  * @returns {Object} - The result after a user has been removed from a channel
  */
 export const removeFromChannel = async (fellowEmail, channel) => addOrRemove(fellowEmail, channel, 'kick');
+
+/**
+ * @function onboardingAutomation
+ * @desc combines all onboarding functions
+ *       into one function.
+ *
+ * @param {string} fellowEmail - user's email
+ * @param {string} channel - the channel id
+ * @param {string} partnerId - the prtner id
+ *
+ * @returns {Object} - With a success message
+ */
+
+export const onboardingAutomation = async (fellowEmail, channel, partnerId) => {
```

Comment:
The variable name `onboardingAutomation` is pretty generic, how about we have `onboardingSlackAutomation`???

Reply:
I think onboardingSlackAutomation is more descriptive and better explains what the function does. Let me change it.